src/ui/components/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,100 @@
+import InputAdornment from "@mui/material/InputAdornment";
+import Tooltip from "@mui/material/Tooltip";
+import { UseFormRegister } from "react-hook-form";
+
+import { PasswordFormFields } from "@src/types";
+import { usePasswordInput } from "@src/ui/components/PasswordInput/usePasswordInput";
+
+import { Icon } from "../Icon";
+import { Input } from "../Input";
+
+import "./password-input.scss";
+
+export interface PasswordInputProps {
+  isShowInfo: boolean;
+  isShowConfirmPassword: boolean;
+  errors: Partial<PasswordFormFields & { root: string }>;
+  register: UseFormRegister<PasswordFormFields>;
```

Comment:
This component doesn't need to know about react-form-hooks, it should have the same props as `Input` has.